### ISSUE TITLE
Wrong centroid calculation

### ISSUE THREAD

**Issue by raviqqe** [Turfjs/turf#1539]
```typescript
import { centroid, polygon } from "@turf/turf";

console.log(centroid(polygon([[[-1, -1], [1, -1], [1, 1], [-1, 1], [-1, -1]]])));
```

The result should be a point at `(0, 0)` but is actually `(-0.2, -0.2)`. The algorithm is using the enclosing point twice.

**Comment by rowanwins** [comment]
Great pickup thanks @raviqqe 

I'll get a fix for this within the v7 branch and add a test.

**Comment by vomc** [comment]
This is probably in development, but I just made a Fiddle using `7.0.0-alpha.1` and the `centroid` still seems off: https://jsfiddle.net/9z37srjb/3/

**Comment by rowanwins** [comment]
Yeah the change hasn't been released yet @vomc , it was added after the `7.0.0-alpha.1` release
